src/palace/manager/api/admin/controller/work_editor.py

```diff
@@ -62,13 +63,13 @@ def details(self, identifier_type, identifier):
 
         :return: An OPDSEntryResponse
         """
-        self.require_librarian(flask.request.library)
+        self.require_librarian(get_request_library())
 
-        work = self.load_work(flask.request.library, identifier_type, identifier)
+        work = self.load_work(get_request_library(), identifier_type, identifier)
         if isinstance(work, ProblemDetail):
             return work
 
-        annotator = AdminAnnotator(self.circulation, flask.request.library)
+        annotator = AdminAnnotator(self.circulation, get_request_library())
```

Comment:
It might make sense to set a variable and use it here, rather than calling the `get_request_library` machinery multiple times here. It also makes it clearer that the value is the same in each case.

I saw this in a few other places in the PR, as well.

Reply:
This was mostly an exercise in find / replace, so i missed these.

I pushed another commit that patched up the most egregious cases of repetition that i saw.

Reply:
I thought as much, but figured it was worth dealing with these now, rather than having to hunt them down later.